x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -306,34 +306,6 @@ describe('Create case', () => {
       });
     });
 
-    it('does not submits the title when the length is longer than 160 characters', async () => {
```

Comment:
Removed this test as e2e already has a test `displays errors correctly while creating a case` in the file `x-pack/test/functional_with_es_ssl/apps/cases/group1/create_case_form.ts`